app/models/webhooks/slack/base.rb

```diff
@@ -0,0 +1,50 @@
+Webhooks::Slack::Base = Struct.new(:event) do
+
+  def username
+    "Cobudget Bot"
+  end
+
+  def icon_url
+    ENV["SLACK_ICON_URL"]
```

Comment:
this should be set to point to a 48x48 px image

SLACK_ICON_URL=https://someimageurl.com/picture.png